### PR TITLE
Add "Whos is online" toast provider for GS

### PR DESCRIPTION
**Pull request by iszmais**
Due to changes made inside the load of toast an provider interface was established to porvide the initial toasts without another request: https://github.com/ILIAS-eLearning/ILIAS/pull/5121

Therefore every provider of toast is obliged to provide a new toast Provider if his service want to add toast to the page.
ATM this only applies to initial toast but will probably be extended to all toast in near future.

This PR is an Proposal for toast inside your Service. Feel free to use and manipulate this template to your fitting and provide a PR for Trunk.
If you dont want to provide Toast with your Service feel free to cloase this PR.

### CODE REVIEW

File: Services/Awareness/GlobalScreen/WhosOnlineToastProvider.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+declare(strict_types=1);
+
+namespace ILIAS\Awarness\GlobalScreen;
+
+use ILIAS\GlobalScreen\Scope\Toast\Provider\AbstractToastProvider;
+use ILIAS\Notifications\Repository\ilNotificationOSDRepository;
+use ILIAS\UI\Component\Symbol\Icon\Standard;
+use ILIAS\Notifications\ilNotificationOSDHandler;
+
+class WhosOnlineToastProvider extends AbstractToastProvider
```
Comment (alex40724) [inline Services/Awareness/GlobalScreen/WhosOnlineToastProvider.php]: WhosOnlineToastProvider -> WhoIsOnlineToastProvider

File: Services/Awareness/GlobalScreen/WhosOnlineToastProvider.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+namespace ILIAS\Awarness\GlobalScreen;
```
Comment (alex40724) [inline Services/Awareness/GlobalScreen/WhosOnlineToastProvider.php]: ILIAS\Awarness\GlobalScreen > ILIAS\Awareness\GlobalScreen

**Review by alex40724 — APPROVED** [review]
Thanks!. Looks good. There are only some minor typos imo. So this works "out of the box" and can simply be merged to release 8 and trunk, right?

File: Services/Awareness/GlobalScreen/WhosOnlineToastProvider.php (hunk 2 above)
Comment (iszmais) [reply Services/Awareness/GlobalScreen/WhosOnlineToastProvider.php]: This typo seemed to have some other occurences therefore i fixed all of them at once.

**Comment by iszmais** [comment]
@alex40724 Yes this can be merged.

**Comment by iszmais** [comment]
Due to major refactoring within the notification process this PR will be closed.
The new approach can be find here https://github.com/ILIAS-eLearning/ILIAS/pull/5730

**Pull request closed without merging**